test/image/compare_pixels_test.js

```diff
@@ -26,12 +26,12 @@ else runSingle(userFileName);
 function runAll () {
     test('testing mocks', function (t) {
 
-        console.error('### beginning pixel comparison tests ###');
         var files = fs.readdirSync(constants.pathToTestImageMocks);
 
         // -1 for font-wishlist and
         // -38 for the gl2d mocks
-        t.plan(files.length - 39);
+        // -1 for gl3d_bunny-hull
```

Comment:
does this mean `files.length -1` for bunny-hull? Perhaps we can expand this comment with a little more description.